packages/gatsby-cli/src/create-cli.js

```diff
@@ -295,6 +299,7 @@ module.exports = argv => {
       global: true,
     })
     .option(`no-color`, {
+      alias: `no-colors`,
```

Comment:
I can remove this, but seems like `chalk` automatically support that as well, so this is just QoL addition (not part of bug fix)

Reply:
I think this is a good idea!